### ISSUE TITLE
sdf_copy_to error

### ISSUE THREAD

**Issue by kennedyshane10** [sparklyr/sparklyr#2357]
---

I am new to using sparklyr. I have successfully connected to a cluster. However, when I try to import a dataset using sdf_copy_to I get the following error:

Error in if (n > 0) c(NA_integer_, -n) else integer() : 
  missing value where TRUE/FALSE needed

The code I am using is as follows:
```r
install.packages("sparklyr")
library(sparklyr)
spark_available_versions()

spark_install(version = "2.2.1")

Sys.setenv(SPARK_HOME="/.../spark/spark-2.2.1")
Sys.getenv("SPARK_HOME")
config <- spark_config()
config$spark.yarn.queue <- "..."

sc <- spark_connect(master="yarn-client", config=config)
spark_version(sc)

library(dplyr)
data(iris)
sdf_copy_to(sc, iris, overwrite=TRUE)
```

**Comment by samuelmacedo83** [comment]
Hi @kennedyshane10 , I think you can fix using this #1829

**Comment by yl790** [comment]
Sounds like this is at least fixed in the master branch of sparklyr. If not, please feel free to re-open this issue (preferably with actual example data that would repro the issue).
